.github/workflows/gh-ci.yaml

```diff
@@ -212,8 +212,16 @@ jobs:
     - name: deploy_docs
       if: github.event_name != 'pull_request'
       run: |
-        # place the deploy call here
-        echo "Oh, maple syrup roast parsnips [Richard Gowers]"
+        python -c 'import os,sys,fcntl; flags = fcntl.fcntl(sys.stdout, fcntl.F_GETFL); fcntl.fcntl(sys.stdout, fcntl.F_SETFL, flags&~os.O_NONBLOCK);'
```

Comment:
I know this has been around forever, but mostly out of curiosity, what does it actually do?

Reply:
I think there used to be a comment somewhere... related to hanging builds? @lilyminium had explained it to me and I promptly forgot.